validation/standalone-constraint-constraint.ttl

```diff
@@ -214,6 +215,142 @@
     ] ;
 .
 
+:Annotation
+    a sh:NodeShape ;
+    sh:targetObjectsOf meta:annotation ;
+    sh:property
+        [
+            sh:path rdf:type ;
+            sh:minCount 1 ;
+            sh:message "annotation needs an rdf:type" ;
+        ],
+        [
+            sh:path meta:annotationContext ;
+            sh:nodeKind sh:BlankNode ;
+        ],
+        [
+            sh:path
+                (
+                    [ sh:inversePath meta:annotation ]
+                ) ;
+            sh:class cube:MeasureDimension ;
+            sh:message "annotated dimension must be a cube:MeasureDimension" ;
+        ] ;
+.
+
+:AnnotationValue
+    a sh:NodeShape ;
+    sh:targetObjectsOf meta:annotation ;
+    sh:message "annotation must have exactly one schema:value or a combination of schema:minValue and schema:maxValue" ;
+    sh:xone
+        (
+            [
+                sh:property
+                    [
+                        sh:path schema:value ;
+                        sh:minCount 1 ;
+                        sh:maxCount 1 ;
+                    ] ;
+            ]
```

Comment:
This `sh:property could be removed if on decides on dropping `schema:value` in favor of `schema:minValue`/`schema:maxValue`.